README.md

```diff
@@ -3,263 +3,441 @@
+
+
+
+###  health_check
+
+| Method  | URI     | Name   | Summary |
+|---------|---------|--------|---------|
+| GET | /api/ocloudNotifications/v2/health | [get health](#get-health) | (Extensions to O-RAN API) Returns the health status of API. |
+
+
+
+###  publishers
+
+| Method  | URI     | Name   | Summary |
+|---------|---------|--------|---------|
+| GET | /api/ocloudNotifications/v2/publishers | [get publishers](#get-publishers) | (Extensions to O-RAN API) Get publishers. |
+
+
+
+###  subscriptions
+
+| Method  | URI     | Name   | Summary |
+|---------|---------|--------|---------|
+| POST | /api/ocloudNotifications/v2/subscriptions | [create subscription](#create-subscription) | Creates a subscription resource for the Event Consumer. |
+| DELETE | /api/ocloudNotifications/v2/subscriptions | [delete all subscriptions](#delete-all-subscriptions) | (Extensions to O-RAN API) Delete all subscriptions. |
+| DELETE | /api/ocloudNotifications/v2/subscriptions/{subscriptionId} | [delete subscription](#delete-subscription) | Delete a specific subscription. |
+| GET | /api/ocloudNotifications/v2/subscriptions/{subscriptionId} | [get subscription by ID](#get-subscription-by-id) | Returns details for a specific subscription. |
+| GET | /api/ocloudNotifications/v2/subscriptions | [get subscriptions](#get-subscriptions) | Retrieves a list of subscriptions. |
+
+
+
+## Paths
+
+### <span id="create-subscription"></span> Creates a subscription resource for the Event Consumer. (*createSubscription*)
 
 ```
-# Pub/Sub Rest API
-Rest API spec .
+POST /api/ocloudNotifications/v2/subscriptions
+```
 
-## Version: 1.0.0
+Creates a new subscription for the required event by passing the appropriate payload.
 
+#### Parameters
 
-### /publishers/
+| Name | Source | Type | Go type | Separator | Required | Default | Description |
+|------|--------|------|---------|-----------| :------: |---------|-------------|
+| SubscriptionInfo | `body` | [SubscriptionInfo](#subscription-info) | `models.SubscriptionInfo` | |  | | The payload will include an event notification request, endpointUri and ResourceAddress. The SubscriptionId and UriLocation are ignored in the POST body (these will be sent to the client after the resource is created). |
 
-#### POST
-##### Summary
+#### All responses
+| Code | Status | Description | Has headers | Schema |
+|------|--------|-------------|:-----------:|--------|
+| [201](#create-subscription-201) | Created | Shall be returned when the subscription resource is created successfully. |  | [schema](#create-subscription-201-schema) |
+| [400](#create-subscription-400) | Bad Request | Bad request. For example, the endpoint URI is not correctly formatted. |  | [schema](#create-subscription-400-schema) |
+| [404](#create-subscription-404) | Not Found | Not Found. Subscription resource is not available. For example, PTP is not supported by the node. |  | [schema](#create-subscription-404-schema) |
```

Comment:
PTP is not supported -- should be subscription not available

Reply:
subscription fro the resource not available

Reply:
removed PTP example